Source/Flow/Private/Nodes/Route/FlowNode_NamedRerouteUsage.cpp

```diff
@@ -29,22 +25,14 @@ void UFlowNode_NamedRerouteUsage::SetNodeName()
 	}
 	else
 	{
-		NodeTitle = FName(TEXT("Invalid named reroute"));
+		NodeTitle = FName(TEXT("Invalid Named Reroute"));
 	}
+	Modify();
```

Comment:
Modify() must be called before any change you want the transaction system to store. Please move Modify() to the beginning of the function body